### DISCUSSION TITLE
自定义模型的配置文件在哪写？

### DISCUSSION THREAD

**Discussion by WillWillWong** [open-compass/opencompass#965] [Q&A]
在文档中找了没找到，想对本地模型进行测试。

**Comment by bittersweet1999** [comment]
If your model is a huggingface model, just follow https://github.com/open-compass/opencompass/blob/main/configs/models/hf_llama/hf_llama2_13b_chat.py to set the path and tokenizer_path to your local path

**Reply by WillWillWong** [reply]
不是hf 模型,**https://opencompass.readthedocs.io/zh-cn/latest/advanced_guides/new_model.html**,After I wrote this, I couldn't find the configuration file

**Comment by bittersweet1999** [comment]
So what exactly is your problem, the config path should be configs/models/your_model/your_model.py

**Reply by WillWillWong** [reply]
/models/your_model/your_model.py ,now i write my model.py,but i don't can run.py mymodel to run

**Reply by bittersweet1999** [reply]
You can understand in this way: there are three configs, model config, dataset config and a overall config, you just write your model config, and you need to set which dataset to be test, so you need to also import a dataset config, and write them all in an overall config,  just like this: https://github.com/open-compass/opencompass/blob/main/configs/eval_demo.py

**Reply by bittersweet1999** [reply]
Actually, there is a video to teach you how to use opencompass, just see in https://www.bilibili.com/video/BV1Gg4y1U7uc/
